src/DripsHub.sol

```diff
@@ -274,7 +275,7 @@ abstract contract DripsHub {
         uint256 userId,
         uint256 assetId,
         SplitsReceiver[] memory currReceivers
-    ) public virtual returns (uint128 collectableAmt, uint128 splitAmt) {
+    ) public whenNotPaused returns (uint128 collectableAmt, uint128 splitAmt) {
```

Comment:
Now, it is way cleaner. Just the modifier here without the wrapper functions :+1: